components/Navbar.vue

```diff
@@ -9,7 +9,10 @@
     <div class="container items-center" :class="{ 'is-fluid': !isTouch }">
       <!-- BRAND -->
       <div class="navbar-brand">
-        <nuxt-link to="/" class="navbar-item logo nuxt-link-active">
+        <nuxt-link
+          to="/"
+          class="navbar-item logo nuxt-link-active"
+          @click="closeAllModals">
```

Comment:
should it be like
```suggestion
          @click.prevent="closeAllModals">
```
?